sourced/ml/cmd/args.py

```diff
@@ -91,3 +91,9 @@ def add_bow_args(my_parser: argparse.ArgumentParser):
     my_parser.add_argument(
         "--batch", default=BOWWriter.DEFAULT_CHUNK_SIZE, type=int,
         help="The maximum size of a single BOW file in bytes.")
+
+
+def add_dzhigurda_arg(my_parser):
+    my_parser.add_argument(
+        "--dzhigurda", default=0, type=int,
+        help="Number of the examined commit in the history. 0 corresponds to HEAD only commits.")
```

Comment:
message is not clear -> change to something more verbose like: 
```
"Number of additional snapshots kept for each document, 
starting from the HEAD. 1 adds HEAD~1, 
2 adds HEAD~1 and HEAD~2, etc. -1 adds all snapshots"
```

Reply:
I do not like `snapshots` prefer to keep `commits`, but I get your point.